### PR TITLE
Corrected grammar pointed out by translators

### PR DESCRIPTION
**Pull request by marek-lach**
The user 'UniqueActive' pointed these issues out on Crowdin.

Thank you!

### CODE REVIEW

File: templates/partials/instance_description.rs.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h2>@i18n!(ctx.1, "About {0}"; instance.name)</h2>
                 @Html(instance.short_description_html)
                 <section class="stats">
                     <div>
-                        <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> people"; n_users))</p>
+                        <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> users"; n_users))</p>
```
Comment (BaptisteGelez) [inline templates/partials/instance_description.rs.html]: I would have preferred to use "people" for both, it feels more like they are real human beings than "users" IMO.

Comment (marek-lach) [reply templates/partials/instance_description.rs.html]: Right. Done :-)

**Comment by marek-lach** [comment]
> Also they saw a `'` that shouldn't be here in templates/users/followed.rs.html IIRC, could you please remove it?

Yeah, I cannot for the life of me find where in that file the incorrect ' symbol actually is... hmmm...

File: templates/instance/about.rs.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h1>@i18n!(ctx.1, "About {0}"; instance.name)</h1>
     <div class="banner">
         <section class="stats">
         <div>
-            <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> users"; n_users))</p>
+            <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> people"; n_users))</p>
```
Comment (igalic) [inline templates/instance/about.rs.html]: 👍

would be easier to remember if `n_users` was `n_people`

Comment (marek-lach) [reply templates/instance/about.rs.html]: It's honestly not a big problem. I think from a programmer's perspective, having the declaration as `users` is easier to understand in code maybe?

Comment (igalic) [reply templates/instance/about.rs.html]: true, true…

**Pull request merged**